pages/risorse.md

```diff
@@ -20,6 +20,7 @@ Ogni utente può aggiungere in qualsiasi momento e autonomamente proprie risorse
 - [Appunti Exyss _(Simone Bianco)_](https://github.com/Exyss/university-notes)
 - [Appunti ph04 _(Alessio Bandiera)_](https://github.com/orgs/ph-notes/repositories)
 - [Appunti Rurik-D _(Emanuele D'Agostino)_](https://github.com/Rurik-D/University-notes)
+- [Appunti CasuFrost (Marco Casu)_](https://github.com/CasuFrost/UNIVERSITA-)
```

Comment:
There'se a problem: you close the _underline_ after `(Marco Casu)_` with an `_`, but you don't open it before the `(`, it should be `_(Marco Casu)_`.